### DISCUSSION TITLE
How do you clear invalid Kong credentials?

### DISCUSSION THREAD

**Discussion by Stokestack** [supabase/supabase#27895] [Questions]
Running Supabase from a Docker container, I entered incorrect credentials when attempting to access the dashboard, which resulted in a page showing
```
Kong Error
Invalid authentication credentials.
```

But there's no affordance provided to try again. Clearing the browser data associated with localhost didn't get rid of this state either. So now the dashboard is inaccessible. What is the expected course of action here?

**Comment by oconr** [comment]
Did you ever find a solution to this? I'm facing the same issue, it works if I use incognito but normal browser keeps saying 'Invalid authentication credentials' even after clearing all the browser data

**Comment by Stokestack** [comment]
Yeah, somebody else filed a similar bug report and there was a workaround in the comments... let me dig it up.

[Here it is.](https://github.com/supabase/supabase/issues/19713)

Hope that helps.

**Comment by ClickFuzz** [comment]
My issue was that I had auth set up in my domain's nginx config file:
/etc/nginx/sites-available/supabase.mydomain.com
